acceptance/tests/loader/func4x_loadable_from_modules.rb

```diff
@@ -80,4 +80,9 @@ class helloworld {
     assert_match(/^Generated, 1 => 10, 2 => 20, 3 => 30$/, stdout, "Generated the wrong content")
   end
 
+  teardown do
```

Comment:
I reverted the commit in PR #3698, added in the other puppet user & group commits, and acceptance passed.